src/App.tsx

```diff
@@ -1,10 +1,61 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo, TodoStatus } from './types/Todo';
+import { ErrorNotification } from './ErrorNotification';
+import { TodosHeader } from './TodosHeader';
+import { TodosList } from './TodosList';
+import { TodosFooter } from './TodosFooter';
 
-const USER_ID = 0;
+const USER_ID = 10332;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [statusFilter, setStatusFilter] = useState<TodoStatus>(TodoStatus.All);
+
+  const loadTodos = async () => {
+    try {
+      const response = await client.get<Todo[]>('/todos?userId=10332');
```

Comment:
```suggestion
      const response = await client.get<Todo[]>('/todos?userId=10332');
```
move `10332` to the variable, do not hardcode it. Also, you already have a function `getTodos` use it
```suggestion
      const response = await getTodos(USER_ID);
```